chipy_org/apps/sponsors/templates/sponsors/sponsor_list.html

```diff
@@ -5,6 +5,10 @@
 {% block head_title %}{% trans "Sponsor List" %}{% endblock %}
 
 {% block body %}
+
+<h2>{{meeting_attendees}} Python Enthusiasts Attended ChiPy Meetings in the Last Year!!</h2>
```

Comment:
Hey Sam.  I think this goes with a different pull request you are working on.

Reply:
Deleting those lines